libguard/guard_interface.cpp

```diff
@@ -391,6 +391,9 @@ void invalidateAll()
     {
         for_each_guard(file, pos, existGuard)
         {
+            std::optional<std::string> physicalPath = getPhysicalPath(existGuard.targetId);
```

Comment:
please use braces code will be more maintainable.
if()
{
}

Reply:
also capture the test results in the commit message, the PR(conversations)might be gone in future but the commit message will stay, you can just use some small snippet.